cves/CVE-2016-6817.yml

```diff
@@ -118,10 +119,10 @@ discovered:
     If there is no evidence as to how this vulnerability was found, then you
     may leave the entries blank except for "answer", BUT please write down
     where you looked in "answer".
-  answer:
-  date:
+  answer: https://www.securityfocus.com/bid/94462 only says "A vendor has reported this issue"
```

Comment:
Thanks for providing a link. Maybe add a brief description for the researchers so they don't have to manually go to the link and read.

Reply:
Description added in new push